Circle.cpp

```diff
@@ -1,14 +1,28 @@
 #ifndef Circle_cpp
 #define Circle_cpp
+
 #include <iostream>
-#include "Draw.cpp"
+#include "Shape.cpp"
+#include "Area.cpp"
+
 using namespace std;
-class Circle : public Draw
+class Circle : public Shape
 {
 public:
     void drawShape() override
     {
-        cout << "circle" << endl;
+        cout << "circle" << endl; 
+    }
+    int inputAndCalculateArea() override
```

Comment:
This function is performing multiple operations, as indicated by the name also.
Taking input of the dimensions of shape should not be part of this function or vice versa.